tests/common.go

```diff
@@ -2533,7 +2536,8 @@ func TearDownBackupRestoreAll() {
 			Uid:  BackupLocationUID,
 		},
 	}
-	ctx, err := backup.GetPxCentralAdminCtx()
+	//ctx, err := backup.GetPxCentralAdminCtx()
```

Comment:
Remove comment

Reply:
Hi Vinayak,
these are px-backup changes and needs some additional changes for it work along with regular longevity.
Hence commented for time being and will be fixed in next sprint